README.rst

```diff
@@ -27,7 +27,8 @@ against K
 ============================
 data/WuExampleExpression.csv
 ============================
-sample dataset
+sample dataset(also available for download `here
```

Comment:
Quick (possibly silly) question: Did you want a space between "sample dataset" and the parenthesis right after it?